aiohttp/pytest_plugin.py

```diff
@@ -55,7 +56,10 @@ def test_client(loop):
 
     @asyncio.coroutine
     def _create_from_app_factory(app_factory, *args, **kwargs):
-        app = app_factory(loop, *args, **kwargs)
+        if not isinstance(app_factory, Application):
+            app = app_factory(loop, *args, **kwargs)
+        else:
+            app = app_factory
```

Comment:
@asvetlov  When I debug the test case, parameter as a create_app function also args and kwargs are both empty.